.github/meta_issue_template/release-1.1.1-plugin-issues-min.md

```diff
@@ -0,0 +1,53 @@
+This is a component issue for release 1.1.1.
+Coming from [release issue 1.1.1](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+
+<details><summary>How to use this component issue</summary>
+<p>
+
+## This Component Issue
+This component issue captures the state of the OpenSearch release, on component/plugin level, its assignee is responsible for driving the release of the component.  Please contact them or @mention them on this issue for help. 
+
+## Release Steps
+There are several steps to the release process, components that are behind present risk to the release.  Component owners resolve tasks on this ticket to communicate with the overall release owner.
+
+Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.
+
+You can find all the corresponding dates of each step in the release issue above.
+
+</p>
+</details>
+
+
+### You can find all the date in above issue 
+### (We make changes to both OpenSearch and OpenSearch-Dashboards for 1.1.1)
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] All the tasks in this issue have been reviewed by the release owner.
+- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.
+
+### CI/CD
+
+- [ ] If including changes in this release, increment plugin version on 1.1 branch to `1.1.1`, otherwise stay on `1.1.0`.
+- [ ] All code changes for 1.1.1 are complete.
+- [ ] Ensure working and passing CI.
+
+### Pre-Release
+
+- [ ] Merge any changes to 1.1 you need for the patch
+- [ ] Complete integration and sanity tests, and update results in the comment, contact corresponding assigner in meta issue above if needed.
```

Comment:
Encouraging direct contact to the release owner isn't going to scale, making sure this process can be followed without any direct contact.  A link to an example would go along way here to make sure folks are following through consistently would be great.

```suggestion
- [ ] Complete integration and sanity tests, and update results in the comment, [example](link).
```

Reply:
I totally get that but at the moment after viewing through several issues I cannot find a good example, as component owners would just check CI results or do manual testing.

Would you think adding my side of the integTest post would help?
https://github.com/opensearch-project/opensearch-build/issues/1118

Reply:
Anyway I will change it with a link to my issue, and extend from there later on.
Thanks.

Reply:
Adding a link to a `RELEASE_PROCESS.MD#HowToPublishTestResults` that we can fill in later works too.

Reply:
Where is this file exist on the repo?
RELEASE_PROCESS.MD#HowToPublishTestResults

Reply:
It doesn't exist yet, but it's a great fast follow

Reply:
Got it.